Danger/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Dangerfile",
+    platforms: [.macOS("10.15")],
+    products: [
+        .library(
+            name: "DangerDepsProduct",
+            type: .dynamic,
+            targets: ["DangerDependencies"]
+        ),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/realm/SwiftLint", from: "0.53.0"),
+        // Danger
+        .package(name: "danger-swift", url: "https://github.com/danger/swift.git", from: "3.17.1"),
```

Comment:
[3.18.0](https://github.com/danger/swift/releases/tag/3.18.0) がリリースされていました

```suggestion
        .package(name: "danger-swift", url: "https://github.com/danger/swift.git", from: "3.18.0"),
```